ci/install.sh

```diff
@@ -3,11 +3,10 @@ set -euxo pipefail
 main() {
     local tag=$(git ls-remote --tags --refs --exit-code https://github.com/rust-lang-nursery/mdbook \
                     | cut -d/ -f3 \
-                    | grep -E '^v0.2.[0-9]+$' \
+                    | grep -E '^v[0-9]+$' \
```

Comment:
```suggestion
                    | grep -E '^v0.[0-9]+.[0-9]+$' \
```

Should probably something like this instead?

Reply:
Happy to.

I was modeling this of the way the same script works in the embedonomicon - should I change it there too?

Reply:
Do you think it would be better to pin it to the 0.3 series, so breaking changes have to be explicitly opted in?

Reply:
OK, I updated to your suggestion. I still think it's worth debating:

- Being more specific -- pinning to 0.3.x instead of 0.x.y
- Doing this same pinning on other `rust-embedded` books

Reply:
OK, I updated to your suggestion. I still think it's worth debating:

- Being more specific -- pinning to 0.3.x instead of 0.x.y
- Doing this same pinning on other `rust-embedded` books

(initial comment was from wrong account; sorry)

Reply:
And the first version had a stupid mistake. Updated again

Reply:
Originally it was free IIRC. We only pinned it to 0.2.1 due to a problem with later versions.